src/cli/run_command.zig

```diff
@@ -557,9 +556,6 @@ pub const RunCommand = struct {
                     }
 
                     Output.prettyErrorln("<r><red>error<r>: Failed to run \"<b>{s}<r>\" due to <r><red>{s}<r>", .{ basenameOrBun(executable), @errorName(err) });
-                    if (@errorReturnTrace()) |trace| {
```

Comment:
Are all of these meant to be replaced with `bun.handleErrorReturnTrace(err, @errorReturnTrace());`? These four seem to be missing if so

Reply:
they are all at line 533